### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by Naviato** [elax46/custom-brand-icons#692] [General]
@elax46 Thank you and everyone else who has contributed to this project. I use HA to manage a large pinball collection and know others who do the same. I'd like to submit a few logos and icons.  Are there any specs to follow? I can provide them as .SVG or any other format. Thanks.

**Comment by elax46** [comment]
You can open a issues @Naviato by specifying the icon request. You can find the guidelines there

**Comment by rchiileea** [comment]
@naviato, I have made a single pinball icon for you, but having simular icons for individual machines would be a no no, if you want I can adjust the icon to show multiple, so there is single and multiple. but I see a single icon as a good way for multiple machines as you can add a custom lable to a card be it number 1-100 or the actual name of the machine.

**Comment by Naviato** [comment]
Thanks for working on this. If you can eliminate the boxes or use one box, similar to what I'm attaching, that would make the image more realistic. 

<img width="186" alt="pins2" src="https://github.com/user-attachments/assets/ce79537e-bed0-4d51-b2ed-6a419cd2da15">
